### DISCUSSION TITLE
Unexpected Error on Setup

### DISCUSSION THREAD

**Discussion by RandomJury** [lkempf/casambi-bt-hass#66] [Q&A]
Added the Integration and my Casambi device was automatically detected in Home Assistant but when I enter my admin password I get an error. Tried and reset account password but even with numerous tries doesnt work. **Which password should be entered in the setup and where to get it?**

![Screenshot 2024-02-22 233824](https://github.com/lkempf/casambi-bt-hass/assets/139177702/d9f100c1-0c2e-4c65-9500-d3b9645fea34)
![Screenshot Casambi-bt error](https://github.com/lkempf/casambi-bt-hass/assets/139177702/f7a5e648-d40e-42fa-a5b4-217153d7f608)

**Comment by Alexei-vk2** [comment]
Faced with this situation:
Casambi integration is installed in the home assistant.
when the "unavailable" condition occurs, the lights turn on and off.

is there any way to make sure that even after the "not available" state, the devices have the last mode so that they are turned off?  
can you tell me how to do this?
![photo_5328170375848255340_y](https://github.com/lkempf/casambi-bt-hass/assets/166062625/9a05cf99-150d-434e-b7eb-8482a1ef6e0b)
